src/qt/rpcconsole.cpp

```diff
@@ -861,17 +861,18 @@ void RPCConsole::clear(bool keep_prompt)
            "Use %3 and %4 to increase or decrease the font size.\n"
            "Type %5 for an overview of available commands.\n"
            "For more information on using this console, type %6.\n"
-           "\n"
-           "%7WARNING: Scammers have been active, telling users to type"
-           " commands here, stealing their wallet contents. Do not use this console"
-           " without fully understanding the ramifications of a command.%8")
+           "\n")
             .arg(PACKAGE_NAME,
                  "<b>" + ui->clearButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>" + ui->fontBiggerButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>" + ui->fontSmallerButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>help</b>",
-                 "<b>help-console</b>",
-                 "<span class=\"secwarning\">",
+                 "<b>help-console</b>") +
+        tr(
+           "%7WARNING: Scammers have been active, telling users to type"
```

Comment:
I think this should be `%1` not `%7` (and `%2` instead of `%8` below) now that it is split to a separate message?